module/VuFind/src/VuFind/Db/Table/ResourceTags.php

```diff
@@ -186,6 +186,84 @@ public function getResourcesForTag($tag, $userId, $listId = null)
         return $this->select($callback);
     }
 
+    /**
+     * Get lists associated with a particular tag.
+     *
+     * @param null|string|array $tag        Tag to match
+     * @param null|string|array $listId     ID of user owning favorite list
+     * @param null|string|array $userId     ID of list to retrieve
+     * @param bool              $publicOnly Whether to return only public lists
+     *
+     * @return \Laminas\Db\ResultSet\AbstractResultSet
+     */
+    public function getListsForTag(
+        $tag, $listId = null, $userId = null, $publicOnly = true
+    ) {
+        $tag = $tag ? (array)$tag : null;
+        $listid = $listId ? (array)$listId : null;
+        $userId = $userId ? (array)$userId : null;
+
+        $callback = function ($select) use (
+            $tag, $userId, $listId, $publicOnly
+        ) {
+            $select->columns(['list_id' => 'list_id', '*']);
```

Comment:
Is there a reason to explicitly specify `list_id` here if you're also specifying `*`? It might also be slightly cleaner to use the `\Laminas\Db\Sql\Select::SQL_STAR` constant.

Reply:
No reason, I took the pattern from `getResourcesForTag`. Fixed and edited to use `SQL_STAR`.

Reply:
Yes, I see that I didn't follow my own advice there; I've opened https://github.com/vufind-org/vufind/pull/1690 to clean that up. :-)